modules/gsp-cluster/data/values.yaml

```diff
@@ -71,6 +71,11 @@ harbor:
   harborAdminPassword: ${harbor_admin_password}
   secretKey: ${harbor_secret_key}
   externalURL: https://registry.${cluster_domain}
+  expose:
+    ingress:
```

Comment:
Is this required?

Reply:
We need to use it in "ingress" mode since the flag totally changes the behaviour of the chart (when set to ClusterIP mode it will spin up an nginx and do a much of other stuff)... so even tho we are not using the `Ingress` record we need the side effects .... setting the ingress hosts is simply to keep the generated stuff sane